include/envoy/stats/tag.h

```diff
@@ -11,7 +13,16 @@ namespace Stats {
 struct Tag {
   std::string name_;
   std::string value_;
+
+  bool operator==(const Tag& other) const {
+    return other.name_ == name_ && other.value_ == value_;
+  };
 };
 
+using TagVector = std::vector<Tag>;
```

Comment:
I think you can drop this one now that this statement is also in stats.h

Reply:
looks like some things include only tag.h and not stats.h, e.g. tag_extractor.h